server/src/main/java/org/opensearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -126,7 +126,7 @@ public class TransportShardBulkAction extends TransportWriteAction<BulkShardRequ
 
     public static final String ACTION_NAME = BulkAction.NAME + "[s]";
 
-    private static final Logger logger = LogManager.getLogger(TransportShardBulkAction.class);
+    private static final Logger logger = Loggers.getLogger(TransportShardBulkAction.class);
```

Comment:
What's the reason for this change? The `Logger getLogger(Class<?> clazz, String... prefixes)` method seems to explicitly disallow this usage.

Side note, seems like it would be better to make that method signature `Logger getLogger(Class<?> clazz, String prefix, String... prefixes)` so you can enforce that constraint at compile time versus runtime.

Reply:
@andrross The change was made with intent to surface underlying Class failure when applying Bulk operation, via `PrefixLogger`

I will close this PR for now; this requires more re-work.